packages/instant/.dogfood.discharge.json

```diff
@@ -1,12 +1,12 @@
 {
     "domain": "0x-instant-dogfood",
-    "build_command": "WEBPACK_OUTPUT_PATH=public yarn build:umd:prod",
+    "build_command": "WEBPACK_OUTPUT_PATH=public yarn build:umd:prod --env.dogfood",
     "upload_directory": "public",
     "index_key": "index.html",
     "error_key": "index.html",
     "trailing_slashes": true,
     "cache": 3600,
-    "aws_profile": "default",
+    "aws_profile": "0xproject",
```

Comment:
I like to be conservative and not have 0x credentials in my default profile -- I think it's good for folks to have a separate "0xproject" aws profile. Being this explicit could help prevent accidents when experimenting or prototyping w/ aws